src/Security/Authentication/test/JwtBearerTests.cs

```diff
@@ -678,7 +678,128 @@ public async Task EventOnChallengeSkip_ResponseNotModified()
             Assert.Empty(response.Response.Headers.WwwAuthenticate);
             Assert.Equal(string.Empty, response.ResponseText);
         }
+        [Fact]
```

Comment:
Empty lines between tests.
```suggestion

        [Fact]
```

Reply:
Updated